lib/src/model/generic_failure.dart

```diff
@@ -13,4 +13,18 @@ class GenericFailure<T> {
 
   @override
   String toString() => "$runtimeType(code: $code, message: \"$message\")";
+
+  @override
+  bool operator ==(dynamic other) =>
```

Comment:
I'd prefer not to use `dynamic` at all (but `Object`).

Reply:
fixed